src/css/workflow.css

```diff
@@ -76,6 +76,10 @@
     align-items: center;
 }
 
+.workflow-node--dash {
+    border-style: dashed;
```

Comment:
Create a common class in base.scss with the name `dc__border-style-dashed`

Reply:
done, and deleted the newly created style class